Source/WebCore/animation/ViewTimeline.cpp

```diff
@@ -74,13 +78,56 @@ Ref<ViewTimeline> ViewTimeline::createFromCSSValue(Style::BuilderState& builderS
     return adoptRef(*new ViewTimeline(nullAtom(), axis, { startInset, endInset }));
 }
 
-// FIXME: compute offset values from options.
+static std::optional<Length> lengthForInset(std::variant<RefPtr<CSSNumericValue>, RefPtr<CSSKeywordValue>> inset)
+{
+    // TODO: Figure out how to convert CSSNumericValue to Length
+    if (holds_alternative<RefPtr<CSSNumericValue>>(inset))
```

Comment:
The vast majority of `holds_alternative()` calls use `std::`, I think we should keep doing this here.